docs/landing-zone-v2/README.md

```diff
@@ -106,10 +118,14 @@ To deploy this Landing Zone you will need to:
     There can only be one organization level ACM policy per organization
 
     ```shell
-    # Validate if an ACM policy exist
+    # enable accesscontextmanager using the current project
+    gcloud services enable accesscontextmanager.googleapis.com --project=${BOOT_PROJECT_ID}
```

Comment:
let's have a chat about this boot project id.. we can discuss it at our weekly call

Reply:
Yes, I think ignoring the current project and moving the enable up above the kcc project create is better